index.js

```diff
@@ -3,35 +3,59 @@
 
 import element from 'magic-virtual-element';
 
+let intervals = {};
+let hoverStates = {};
+
 export default {
-  render: function ({ props }) {
+  render: function (component) {
+    const {state, props} = component;
+    let style;
+    if (state.offset) {
+      style = 'left: -' + state.offset + 'px;';
+    } else {
+      style = 'transition: none; left: 0;';
+    }
+
+    function hover () {
+      hoverStates[component.id] = true;
+    }
+
+    function blur () {
+      hoverStates[component.id] = false;
+    }
+
     return (<div class='news-ticker'>
       <span class='news-ticker__label'>{ props.label || 'The Latest News'}</span>
-      <span class='news-ticker__slider'>
+      <span class='news-ticker__slider' style={style} onMouseOver={hover} onMouseOut={blur}>
         {props.children}
       </span>
     </div>);
   },
   afterMount: function (component, el, setState) {
+    const {props} = component;
     const slider = el.querySelector('.news-ticker__slider');
 
-    const interval = setInterval(function () {
-      slider.style.transition = '';
-      slider.style.left = -slider.firstChild.offsetWidth + 'px';
+    slider.addEventListener('transitionend', function (event) {
```

Comment:
@ellell when we talked you said something about this not working - but I guess you got it working now anyway?